Xamarin.Forms.Core/SwipeView.cs

```diff
@@ -0,0 +1,742 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Input;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[ContentProperty("Content")]
+	[RenderWith(typeof(_SwipeViewRenderer))]
+	public class SwipeView : ContentView, ISwipeViewController, IElementConfiguration<SwipeView>
```

Comment:
either inherit from Layout<View>, or use MenuItem (in that case, ContentView is ok)